Add typescript types and onboarding ping (#525)
MindscapeHQ/raygun4js: CHANGELOG.md

- Unused normalize.css file
  
-->

> ## [3.1.1]

Reviewer: Should be 3.1.0

> ### Added 

Reviewer: Add a change log to note we have new alias option for enableRealUserMonitoring

- Unused normalize.css file
  
-->
## [3.1.0]

### Added 

> - Added a ping to tell Raygun that raygun4js has been set up correctly, this information will be used to help users when setting up a new application. This can be disabled using the new `sendPing` option e.g., `rg4js('sendPing', false)`. The ping contains the version of Raygun4js being used, aswell as what products that have been enabled (CrashReporting and or Real User Monitoring) 
> - Moved the typescript types into the main raygun4js repo
> - Added an new option for enabling Real user monitoring 

Reviewer 2: ```suggestion
- Added a ping to tell Raygun that raygun4js has been set up correctly. The ping is enabled by default but can be disabled using the new `sendPing` option e.g., `rg4js('sendPing', false)`. The ping contains the version of Raygun4js and which products have been enabled (CrashReporting and/or Real User Monitoring). This information will be used to help users when setting up a new application.
- Moved the typescript types into the main raygun4js repo
- Added a new option for enabling Real User Monitoring. `rg4js('enableRealUserMonitoring', true);`
```
